spring-boot-project/spring-boot-parent/pom.xml

```diff
@@ -22,8 +22,8 @@
 		<java.version>1.8</java.version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		<aether.version>1.0.2.v20150114</aether.version>
-		<maven.version>3.1.1</maven.version>
+		<aether.version>1.1.0</aether.version>
+		<maven.version>3.2.5</maven.version>
```

Comment:
Please limit the scope of your PR. Those dependencies upgrade shouldn't be part of this commit. If they are necessary please explain why

Reply:
Maven documentation [recommends](https://maven.apache.org/plugin-developers/plugin-testing.html) to use `maven-plugin-testing-harness` library as a simple way for creating integration tests for Mojo's. `maven-plugin-testing-harness` support maven starting from version `3.2.5`. Updating maven's version triggers updates of dependent libraries. 

I agree that this is should be another PR. I tried to make minimal set of version changes in this  PR related to maven. Actually I was thinking about creating a separate PR with maven 3.5.0 support and clean up maven-related library "hell" there but I am not sure maybe you have strong reasons to support maven from version 3.2.5. If you need it I can create a separate ticket/PR for this and postpone current PR.